nautobot/core/apps/__init__.py

```diff
@@ -39,7 +39,7 @@ def create_or_check_entry(grouping, record, key, path):
         grouping[key] = record.initial_dict
     else:
         for attr, value in record.fixed_fields:
-            if grouping[key][attr] != value:
+            if grouping[key][attr] != value and attr != "weight":
```

Comment:
Might as well just remove `weight` from `fixed_fields` altogether?

Reply:
of course, my bad